.github/workflows/pre-release.yaml

```
@@ -0,0 +1,65 @@
+# Workflow to fetch the latest commit hash on the main branch and upload artifacts to CF storage.
+name: Build and upload from main
+on:
+  workflow_dispatch:
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  define-matrix:
+    name: Define Matrix
```

```suggestion
  stellar-contract-names:
    name: Get all Stellar contracts
```
define-matrix doesn't make much sense